tests/integration_tests/build/test_style.py

```diff
@@ -66,14 +69,15 @@ def test_python_style():
     ])
 
 
-@pytest.mark.skipif(
-    platform.machine() != "x86_64",
-    reason="no need to test it on multiple platforms"
+@pytest.mark.parametrize(
```

Comment:
Can we add a short explanation here or in the commit message why we run clippy for both targets?

Reply:
Here I am rather fixing a bug since we were not running clippy on gnu code (see seccomp crate) so I will mention it in the commit message.

Reply:
Fixed, mentioned it in the commit title.